caffe2/onnx/backend.cc

```diff
@@ -411,6 +426,25 @@ Caffe2Ops Caffe2Backend::CreateConvePoolOpBase(const ModelProto &init_model,
     attr->set_i(1);
   }
 
+  if (attributes.HasAttribute("auto_pad")) {
+    auto auto_pad = attributes.get<const std::string*>("auto_pad");
```

Comment:
`CHECK(auto_pad);`?

Reply:
Do we have a `CHECK` macro? I didn't quite get this.